profiler/profiler.oql/test/unit/src/org/netbeans/modules/profiler/oql/engine/api/impl/OQLEngineTest.java

```diff
@@ -515,6 +516,7 @@ public boolean visit(Object o) {
                 return false;
             }
         });
+        output[0] = output[0].replaceAll("UID=301077366599181[0-9]*", "UID=301077366599181567");
```

Comment:
Could you please add a comment what this making replacement does? Not so much for the change itself, as for future readers, that wonder what was this meant to say.

Reply:
According to @thurka there was some bug in the serialization that affects the `UID` value. Tomáši, can you share the JDK bug reference?

Reply:
https://netbeans.org/bugzilla/show_bug.cgi?id=256761. JDK bug is already fixed, there is no need for workaround.